feature-libs/product-configurator/rulebased/components/show-more/configurator-show-more.component.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  Input,
+  ViewChild,
+} from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Component({
+  selector: 'cx-configurator-show-more',
+  templateUrl: './configurator-show-more.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConfiguratorShowMoreComponent implements AfterViewInit {
+  showMore$ = new BehaviorSubject<boolean>(false);
```

Comment:
`showMore` and `showHiddenText` can be just boolean, right?

Reply:
Picked

Reply:
Done